swarmrl/trainers/episodic_trainer.py

```diff
@@ -79,9 +83,22 @@ def perform_rl_training(
             )
             for episode in range(n_episodes):
                 # Check if the system should be reset.
-                if episode % reset_frequency == 0 or killed:
+                if episode % reset_frequency == 0 and reset_frequency > 0 or killed:
```

Comment:
Can it be true that `reset_frequency == 0` but `episode % reset_frequency == 0`? I don't know for sure, I haven't tested it, I would just be surprised if the remainder of infinity could be defined. In general, what issue prompted this addition to the loop condition?

Reply:
True, that's nonsense. 
Changed it such that it checks if `episode == 0 `.